rpcs3/module_verifier.hpp

```diff
@@ -0,0 +1,82 @@
+#pragma once
+#ifdef _WIN32
+
+#ifndef WIN32_LEAN_AND_MEAN
+#define WIN32_LEAN_AND_MEAN
+#endif
+
+#include <Windows.h>
+
+#include <util/types.hpp>
+#include <Utilities/StrUtil.h>
+#include <Utilities/StrFmt.h>
+
+[[noreturn]] void report_fatal_error(std::string_view);
+
+// Validates that system modules are properly installed
+// Only relevant for WIN32
+class WIN32_module_verifier
+{
+	struct module_info_t
+	{
+		std::wstring_view name;
+		std::string_view package_name;
+		std::string_view dl_link;
+	};
+
+	const std::vector<module_info_t> special_module_infos = {
+		{ L"vulkan-1.dll", "Vulkan Runtime", "https://sdk.lunarg.com/sdk/download/latest/windows/vulkan-runtime.exe" },
+		{ L"msvcp140.dll", "C++ Redistributable for Visual Studio 2015", "https://www.microsoft.com/en-us/download/details.aspx?id=48145" },

```

Comment:
We distribute the VS2019 ones on the guide

https://aka.ms/vs/16/release/VC_redist.x64.exe

Reply:
Noice. Will update the links.

Reply:
Done